src/Notifications/NotificationsViewExtension.cs

```diff
@@ -115,6 +117,12 @@ public void Loaded(ViewLoadedParams viewStartupParams)
         private void LoadNotificationCenter()
         {
             var dynamoView = viewLoadedParams.DynamoWindow as DynamoView;
+
+            if (notificationCenterController != null)
+            {
+                notificationCenterController.Dispose();
+                notificationCenterController = null;
```

Comment:
why?

Reply:
tests (not sure of can happen live) will sometimes trigger the Loaded event multiple times (i.e notificationCenterController = new NotificationCenterController() on every Load event)

It would cause NotificationCenterControllers to be cleaned up by the GC.
Hopefully this will reduce the memory footprint during tests and make it easier to identify where async vs dispose issues are coming from.

Reply:
weird... so the tests trigger the ViewExtensionManager Loaded event multiple times?